fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/commands/InsertIntoTableCommand.java

```diff
@@ -101,6 +105,14 @@ public void setJobId(long jobId) {
 
     @Override
     public void run(ConnectContext ctx, StmtExecutor executor) throws Exception {
+        runInternal(ctx, executor);
+    }
+
+    public void statefulRun(ConnectContext ctx, StmtExecutor executor) throws Exception {
```

Comment:
Looks like `statefulRun()` is same as `run()`?

Reply:
> Looks like `statefulRun()` is same as `run()`?
    change to
    void runWithStatistic(ConnectContext ctx, StmtExecutor executor, LoadStatistic loadStatistic);